lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -107,6 +107,7 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
         return null;
       }
       bitmap = BitmapFactory.decodeByteArray(data, 0, data.length, opts);
+      bitmap = Utils.resizeBitmapIfNeeded(bitmap, asset.getWidth(), asset.getHeight());
```

Comment:
Could you store the bitmap on the line above to a local variable?

Reply:
@gpeal I updated, please check